packages/psd/src/sections/LayerAndMaskInformation/index.ts

```diff
@@ -105,24 +105,25 @@ export function parseLayerAndMaskInformation(
         groupId: alreadyViewed,
         parentGroupId: currentGroupId,
         layerRecord,
+        channels,
       });
       orders.push("G");
     } else if (dividerType === GroupDivider.BoundingSection) {
       // Indicates the end of a layer group
-
       const frame: Frame | undefined = stack.pop();
       if (frame === undefined) {
         throw new PanicFrameStackUnmatched();
       }
 
       const groupId = frame.groupId > 0 ? frame.groupId : undefined;
       const _layerRecord = frame.layerRecord || layerRecord;
-
+      const _channels = frame.channels || channels;
```

Comment:
Why this alternative? For `LayerFrame` we use `channels` directly. I don't think `frame` contains `channels` property anyway.

Reply:
are you saying that frame.channels should not be there?  in previous block we do stack.push({layerRecord}) (stack is Frame). we need corresponding channels, so I expanded Frame so it contains channels too